### ISSUE TITLE
Seasonally themed icons

### ISSUE THREAD

**Issue by Firstyear** [kanidm/kanidm#1596]
We have seasonal images already available - I think we should display them automatically in certain date ranges such as the christmas Kani.

**Comment by yaleman** [comment]
As long as there's an "out" for folks without a sense of humor or particular cultural sensitivities... or both.

**Comment by Firstyear** [comment]
Yeah, this was an "future fun idea" so I'm not in a rush to do it. We'll need a way to allow custom icons in future too anyway.

**Comment by tumbl3w33d** [comment]
> We'll need a way to allow custom icons in future too

Will this be the issue about theming/customizing the look in general (logo, title, headline, link to data privacy terms,…)? Was taking a look around but didn't see any other issue about it. #1955 is somewhat related since it's all about the enduser experience, building trust, make them feel like they're entering the right place.

**Comment by yaleman** [comment]
Please create a new issue specifically for those things, #1955 is specific to the OAuth2 login flow. 

What it looks like you're asking for is admin-configurable theming, which is separate again to both this issue and that one.
